javascript-typing-tutor/javascript-typing-tutor/styles.css

```diff
@@ -0,0 +1,38 @@
+div {
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+h1 {
+  padding: 33px;
+  margin: 0;
+  font-family: monospace;
+}
+
+span {
+  font-size: 3em;
+  display: inline-block;
+  border-bottom: 7px solid white;
+  white-space: pre;
+}
+
+.u {
```

Comment:
this is more of a nit picky thing... and even though I understand what these classes mean, it's better to give them human-readable names like `.underline` or `.green` and `.red` 

coming back to your code later on down the line you'll likely forget what those abbreviated class names mean and makes code harder to read for others as well.